packages/camera/camera_windows/windows/camera.cpp

```diff
@@ -16,6 +16,25 @@ constexpr char kVideoRecordedEvent[] = "video_recorded";
 constexpr char kCameraClosingEvent[] = "camera_closing";
 constexpr char kErrorEvent[] = "error";
 
+// Camera error codes
+constexpr char kCameraAccessDenied[] = "CameraAccessDenied";
+constexpr char kCameraError[] = "camera_error";
```

Comment:
maybe `camera_error_others`?

Reply:
The existing plugin already uses `camera_error` as its fallback error code. Changing this to `camera_error_others` would be a breaking change, which I'd rather do in a separate pull request since this one focuses on introducing camera permission errors.

Also, if we were to do this breaking change I'd prefer to go to a standardized fallback error code. None exists today, but perhaps `CameraError` would be better aligned to the other standardized camera error code?